### PR TITLE
Added rabbitmq icon support

### PR DESCRIPTION
**Pull request by brendandebeer**
Please review the inclusion of rabbitmq icon support in the viasualization - thanks !

### CODE REVIEW

File: src/main/kotlin/com/github/chriskn/structurizrextension/internal/icons/IconRegistry.kt [hunk 1]
```diff
@@ -82,7 +82,8 @@ object IconRegistry {
         "postgresql" to "${GILBARBARA_ICON_URL}postgresql.puml",
         "react" to "${GILBARBARA_ICON_URL}react.puml",
         "redis" to "${GILBARBARA_ICON_URL}redis.puml",
-        "kafka" to "${GILBARBARA_ICON_URL}kafka.puml",
+        "kafka" to "${GILBARBARA_ICON_URL}rabbitmq.puml",
```
Comment (chriskn) [inline src/main/kotlin/com/github/chriskn/structurizrextension/internal/icons/IconRegistry.kt:85]: The icon paths for kafka and rabbitmq are swapped.

**Comment by chriskn** [comment]
Hi @brendandebeer, thanks for the change. I will merge it when the paths are fixed

File: src/main/kotlin/com/github/chriskn/structurizrextension/internal/icons/IconRegistry.kt (hunk 1 above)
Comment (brendandebeer) [reply src/main/kotlin/com/github/chriskn/structurizrextension/internal/icons/IconRegistry.kt]: apologies for the embarrassing mistake - BTW : thanks for the efforts with the extensions ;-)

Comment (chriskn) [reply src/main/kotlin/com/github/chriskn/structurizrextension/internal/icons/IconRegistry.kt]: No problem, you are welcome 👍

**Pull request merged**